build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -170,4 +170,4 @@
           securityContextConfig: legacy
         updateStrategy:
           registryPoll:
-            interval: 1m
+            interval: 5m
```

Comment:
I find that the index image can get into a loop with a really short interval. In testing I found setting this to 5 minutes tended to avoid that constant update loop.